### PR TITLE
add exclude_modeladmins option

### PR DESCRIPTION
**Pull request by PetrDlouhy**

### CODE REVIEW

File: test_project/main/fixtures/django_admin_smoke_tests.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[
```
Comment (SeanHayes) [inline test_project/main/fixtures/django_admin_smoke_tests.json:1]: Is this file needed?

**Comment by SeanHayes** [comment]
Sorry for not getting to this sooner. Did you still want this merged?

**Comment by PetrDlouhy** [comment]
No, this is replaced by #8 which contains all commits from here. I am still working on improving the PR (increasing the coverage a bit more and cleaning some details including that fixture), so if you wait few minutes, you can merge it in better shape. I will write on #8 after I am finished.
